Bottom: c129a1c3d15e0f038d8f92018d8c319a6e035761
Top:    f7ac833393efb34184f8ec2f6235df597ea882f8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-09-30 01:16:15 +0100

Ensure "Gathering" mode is set after painting the screen.



---

diff --git a/src/draw.c b/src/draw.c
index 8322879..f710567 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -2711,4 +2711,6 @@ hid_expose_callback (HID * hid, BoxType * region, void *item)
   Output.fgGC = savefg;
   Output.bgGC = savebg;
   Output.pmGC = savepm;
+
+  Gathering = True;
 }
